README.md

```diff
@@ -189,6 +189,17 @@ SIG-specific GitHub discussions.
 | <a id="sig-contributor-experience"></a>Contributor Experience | Monday alternating between 10:00 PT and 17:00 UTC+8 | [Google Doc](https://docs.google.com/document/d/1CTQI0p3QF8JP8reV8z_ggcs8KE5YVPpQGvAQknw4qP0) | [#otel-contributor-experience](https://cloud-native.slack.com/archives/C06TMJ2R0SK) | [calendar-contributor-experience](https://groups.google.com/a/opentelemetry.io/g/calendar-contributor-experience) | [Juraci Paixão Kröhling](https://github.com/jpkrohling) |
 | <a id="sig-devex"></a>Developer Experience | Wednesday 11:00 PT and 17:00 UTC+8 | [Google Doc](https://docs.google.com/document/d/1QDZYAvGJbqgodJaTxRPmRZh-fEJjb6XstvPUst6h50w) | [#otel-devex](https://cloud-native.slack.com/archives/C01S42U83B2) | [calendar-developer-experience](https://groups.google.com/a/opentelemetry.io/g/calendar-developer-experience) | [Austin Parker](https://github.com/austinlparker) |
 
+### Localization Teams (part of SIG Communications)
+
+| Name | Meeting Time | Meeting Notes | Slack Channel | Meeting Invites Group | [Governance Committee](./community-members.md#governance-committee) Liaison |
+|------|--------------|---------------|---------------|-----------------|--------------------------------|
+| <a id="sig-localization-bn"></a>Bengali (bn) |  |  | [#otel-localization-bn](https://cloud-native.slack.com/archives/C08TBCSAY1F) |  | [Severin Neumann](https://github.com/svrnm) |
+| <a id="sig-localization-zhcn"></a>Chinese (zh-CN) |  |  | [#otel-localization-zhcn](https://cloud-native.slack.com/archives/C08SSK25Y7L) |  | [Severin Neumann](https://github.com/svrnm) |
+| <a id="sig-localization-fr"></a>French (fr-FR) |  |  | [#otel-localization-fr](https://cloud-native.slack.com/archives/C07THD60YLF) |  | [Severin Neumann](https://github.com/svrnm) |
+| <a id="sig-localization-ja"></a>Japanese (ja-JA) | Every thirds Wednesday at 01:00 JST | [Google Doc](https://docs.google.com/document/d/1venl18Obvp3vD-lR_9Hj9k826s_weF0vzWe-TdaDkuY) | [#otel-localization-ja](https://cloud-native.slack.com/archives/C08SGPBN44E) | [calendar-ja-ja-localization](https://groups.google.com/a/opentelemetry.io/g/calendar-ja-ja-localization) | [Severin Neumann](https://github.com/svrnm) |
```

Comment:
nit: s/thirds/third/ ? also adding `(monthly)` would reduce confusion.

Reply:
thanks, fixed via @tiffany76's comment. I kept the (monthly) out since the calendar invite makes this obvious, hope that'S OK